modules/private-clusters-about.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-private-cluster.adoc
+
+[id="private-clusters-about_{context}"]
+= About private clusters
+
+By default, {product-title} is provisioned using publicly-accessible DNS and endpoints. You can set the DNS, Ingress Controller, and API server to private after you deploy your cluster.
+
+[discrete]
+[id="private-clusters-about-dns_{context}"]
+== DNS
+
+If you install {product-title} on installer-provisioned infrastructure, the installation program creates records in a pre-existing public zone and, where possible, creates a private zone for the cluster’s own DNS resolution. In both the public zone and the private zone, the installation program or cluster creates DNS entries for `*.apps`, for ingress, and `api`, for the API server.
```

Comment:
s/ingress/Ingress 
s/ingress controller/Ingress Controller 

Applies to a few places.